homeassistant/components/comelit/manifest.json

```diff
@@ -7,6 +7,6 @@
   "integration_type": "hub",
   "iot_class": "local_polling",
   "loggers": ["aiocomelit"],
-  "quality_scale": "silver",
+  "quality_scale": "platinum",
```

Comment:
Please don't change the quality scale or record in the same PR as other changes.

Reply:
Done